src/operation/aggregate/test.rs

```diff
@@ -188,59 +186,6 @@ async fn op_selection_criteria() {
     });
 }
 
-#[cfg_attr(feature = "tokio-runtime", tokio::test)]
-#[cfg_attr(feature = "async-std-runtime", async_std::test)]
-async fn handle_success() {
```

Comment:
Rather than continuing to maintain these tests that break a lot and don't offer much additional coverage over our integration tests, I decided just to delete the ones that broke. We discussed this a bit in slack, but in retrospect these tests are probably far too close to the actual implementation and are probably more effort than they're worth. Thoughts?

Reply:
agreed, I don't think these are particularly valuable

Reply:
SGTM.